pubspec.yaml

```diff
@@ -0,0 +1,7 @@
+name: plus_plugins_workspace
+
+environment:
+  sdk: ">=2.12.0 <3.0.0"
```

Comment:
Don't we need at least 2.18 here for Melos 3.0.0?

Reply:
I think you wrote this comment a matter of seconds after I pushed a commit that updated it to 2.18.0 😄

Reply:
Seems like yes